.github/workflows/update.yaml

```diff
@@ -114,7 +114,10 @@ jobs:
 
       - name: CS fix SDK
         working-directory: ./sdk
-        run: composer cs-fix
+        run: |
+          composer cs-fix
+          rm .php-cs-fixer.cache # Remove cache for the second run
+          composer cs-fix # Run twice to ensure all large files are fixed
```

Comment:
As far as I understand it is added because of the bug in cs-fixer? Is it a temporary measure?

Reply:
Yes, I'll create a separate issue for generator itself to track it down later

Reply:
Maybe you can leave a comment here as well why it run twice, otherwise it is not clear

Reply:
Fair, added